mm_cataloguer/index_netcdf.py

```diff
@@ -59,6 +59,8 @@
 import functools
 from argparse import ArgumentParser
 
+from dateutil.relativedelta import relativedelta
```

Comment:
``dateutil`` is an undeclared dependency and should be added to `setup.py` and `requirements.txt`.